lambda_functions/archive_service/get_closed_items.py

```diff
@@ -21,7 +21,7 @@ def get_closed_items(event, context):
             print("\n \n \n Getting items \n \n \n")
             allow_all_origins = False
 
-            if 'queryStringParameters' in event and 'url' in event['queryStringParameters']:
+            if 'queryStringParameters' in event and isinstance(event['queryStringParameters'], dict) and 'url' in event['queryStringParameters']:
```

Comment:
Jetzt verstehe ich den `try/except` Ansatz... Ist wohl auch ein gebräuchliches Pattern in Python https://docs.python.org/2/glossary.html#term-eafp